config/routes.rb

```diff
@@ -3,4 +3,5 @@
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   root 'recipes#index'
+  get 'categories', to: 'categories#create'
```

Comment:
This isn't being used, but it should be a `post `instead of a `get`
```suggestion
  post 'categories', to: 'categories#create'
```

Reply:
Should just remove it